.github/workflows/email.yml

```diff
@@ -0,0 +1,83 @@
+name: email
+
+on:
+  push:
+    branches:
+      - main 
+      
+jobs:
+  
+  
+  send_email:
+    runs-on: ubuntu-latest
+  
+    steps:
+   
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+          
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v32
+        with:
+          separator: ","
+         
+          
+      - name: Send mail  
+       
+        if: github.event_name == 'pull_request' && github.event.action == 'closed' && github.event.pull_request.merged == true  
+        uses: dawidd6/action-send-mail@v3
+        with:
+   
+    # Required mail server address if not connection_url:
+           server_address: ${{ secrets.SMTP_PROVIDER}}
+           server_port: 465
+    # Optional whether this connection use TLS (default is true if server_port is 465)
```

Comment:
Can you keep the comment indentation the same as the code indentation and remove some of the empty newlines (might be a style preference, but I find the indentation and extra whitespace make it harder for me to parse)

Reply:
Fixed